src/sentry/static/sentry/app/stores/configStore.jsx

```diff
@@ -43,15 +46,13 @@ const ConfigStore = Reflux.createStore({
         // e.g. unencoded "%"
       }
 
-      // Priority:
-      // "?lang=en" --> user configuration options --> django request.LANGUAGE_CODE --> "en"
-      setLocale(
-        queryString.lang || config.user.options.language || languageOverride || 'en'
-      );
-    } else if (languageOverride) {
-      setLocale(languageOverride);
+      languageCode = queryString.lang || config.user.options.language || languageCode;
```

Comment:
TODO: see if we can also leverage [`navigator.language`](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language) as a last resort.